web/src/lib/components/album-page/album-viewer.svelte

```diff
@@ -148,14 +122,7 @@
         {album.albumName}
       </h1>
 
-      <!-- ALBUM SUMMARY -->
-      {#if album.assetCount > 0}
-        <span class="my-4 flex gap-2 text-sm font-medium text-gray-500" data-testid="album-details">
-          <p class="">{getDateRange()}</p>
-          <p>·</p>
-          <p>{album.assetCount} items</p>
-        </span>
-      {/if}
+      <AlbumSummary assetCount={album.assetCount} {album} />
```

Comment:
it looks like you don't pass in assets here so does this mean it won't show ranges anymore? it seems like this doesn't work the way it used to.

Reply:
It uses the range from the album and should behave exactly as before